cuqi/array/_array.py

```diff
@@ -1,108 +1,266 @@
-import numpy as np
 from cuqi.geometry import _DefaultGeometry1D
 
+# Import the backend module directly to avoid circular import
+import os
+_backend_name = os.getenv("CUQI_ARRAY_BACKEND", "numpy").lower()
```

Comment:
It would be good if users could set the backend diretly in the script using e.g. cuqi.array.set_backend("pytorch") or similar. This may mean rethinking the class creation idea for handing the CUQIarray.